### ISSUE TITLE
Fibonacci versioning

### ISSUE THREAD

**Issue by orsinium** [dephell/dephell#233]
Increment every next version by Fibonacci number: [fibver.org](https://fibver.org/)

1. Add it into [dephell_versioning](https://github.com/dephell/dephell_versioning) with docs and tests. Allow only `major.minor.patch`
1. Update docs in DepHell for `project bump`

**Comment by hugovk** [comment]
> So “Product 55.0.0” versus “Product 34.0.0” sounds much more expensive and feature-full than boring upgrade from “Product 8.0” to “Product 9.0”. 

I disagree and think it may cause more confusion. People were confused when pip first switched to calver in 2018, because of the leap from 10.x to 18.x: "This is very strange. Did the pip project just upgrade from version 10 to version 18? Why?"

* https://stackoverflow.com/questions/51476300/why-did-pip-upgrade-from-version-10-to-version-18

> Moreover, sucessive Fibonacci numbers differs greatly. It is much harder to confuse versions 1.987.2584 and 1.2584.987 instead of 1.15.17 and 1.17.15.

1.987.2584 and 1.2584.987 are much less readable for me. I'd hate to have to say it out loud, or even remember which version is which!

I don't recall a situation confusing something like 1.15.17 and 1.17.15. 

FibVer reminds me a bit of html5lib's asymptotic versioning: https://github.com/html5lib/html5lib-python/issues/282.

**Comment by orsinium** [comment]
> FibVer reminds me a bit of html5lib's asymptotic versioning

html5lib represented in [examples](https://dephell.readthedocs.io/cmd-project-bump.html#projects-that-use-these-versioning-schemes) for ZeroVer.

> I disagree and think it may cause more confusion.

Of course, it's an awful way to version your project. As Roman versioning. Don't treat it seriously.
